canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -455,12 +457,15 @@ class PropertiesMain extends React.Component {
 		}
 	}
 
-	detectResize() {
-		// only hide resize button if resize wasn't from clicking resize button
-		if (!this.resizeClicked) {
-			this.setState({ showResizeBtn: false });
+	detectResize(_width, height) {
```

Comment:
Any reason you have _width and not just `width` to match height?

Reply:
since `width` is not used anywhere in the function build was failing due to unused variable error so to overcome it I included it `_`.